templates/social.mustache

```diff
@@ -1,9 +1,9 @@
 <ul class="social-side">
   <li>
-    <a class="header-mobile github" href="https://github.com/canjs/canjs" target="_blank"><img class="social-icon-small" src="../images/github.png">Github</a>
+    <a class="header-mobile github" href="https://github.com/canjs/canjs" target="_blank"><img class="social-icon-small" src="/docs/images/github.png">Github</a>
```

Comment:
never use absolute references in bit-docs.  You don't know where the site might be hosted.  This should be made relative similar to how links are made relative.